crates/hyperswitch_domain_models/src/business_profile.rs

```diff
@@ -659,6 +659,14 @@ impl From<BusinessProfileSetter> for BusinessProfile {
 }
 
 impl BusinessProfile {
+    pub fn get_is_tax_connector_enabled(&self) -> bool {
+        let is_tax_connector_enabled = self.is_tax_connector_enabled;
+        match &self.tax_connector_id {
+            Some(_id) => is_tax_connector_enabled,
+            _ => false,
+        }
+    }
```

Comment:
[is_some_and](https://doc.rust-lang.org/std/option/enum.Option.html#method.is_some_and) can probably help simplify this.

Reply:
@racnan this function will be refactored later, while introducing tax_connector_id validation